### PR TITLE
chore: prepare release react-northstar 0.66.1

### PR DESCRIPTION
**Pull request by jurokapsiar**
Prepare @fluentui/react-northstar 0.66.1 release

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a2a5de47a528f32fd134f076f06f11d814aa760f'>a2a5de47a528f32fd134f076f06f11d814aa760f</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

File: apps/public-docsite-v9/package.json [hunk 1]
```diff
@@ -25,8 +25,11 @@
     "@fluentui/react-migration-v8-v9": "^9.1.2",
     "@fluentui/react-migration-v0-v9": "9.0.0-alpha.0",
     "@fluentui/react": "^8.105.5",
-    "@fluentui/react-northstar": "^0.66.0",
-    "@fluentui/react-icons-northstar": "^0.66.0",
+    "@fluentui/react-northstar": "^0.66.1",
+    "@fluentui/react-icons-northstar": "^0.66.1",
+    "@fluentui/react": "^8.105.4",
+    "@fluentui/react-northstar": "^0.66.1",
```
Comment (silviuaavram) [inline apps/public-docsite-v9/package.json:31]: This one at the next are also on 28 and 29.

File: apps/public-docsite-v9/package.json [hunk 2]
```diff
@@ -25,8 +25,11 @@
     "@fluentui/react-migration-v8-v9": "^9.1.2",
     "@fluentui/react-migration-v0-v9": "9.0.0-alpha.0",
     "@fluentui/react": "^8.105.5",
-    "@fluentui/react-northstar": "^0.66.0",
-    "@fluentui/react-icons-northstar": "^0.66.0",
+    "@fluentui/react-northstar": "^0.66.1",
+    "@fluentui/react-icons-northstar": "^0.66.1",
+    "@fluentui/react": "^8.105.4",
```
[the remark this answers is not in the document]
Comment (jurokapsiar) [reply apps/public-docsite-v9/package.json:30]: oops wrong merge

**Review by silviuaavram — APPROVED** [review]

**Review by YuanboXue-Amber — APPROVED** [review]

**Review by miroslavstastny — APPROVED** [review]

**Review by ling1726 — APPROVED** [review]

**Pull request merged**